Graphs/TreeGraph/images.js

```diff
@@ -0,0 +1,11 @@
+export * from '@/images/vport';
```

Comment:
I would prefer to avoid adding new images in the graphs! Any chance we can pass and use the base64 for the images?